7/Dockerfile

```diff
@@ -152,7 +153,10 @@ RUN set -xe; \
       docker-php-ext-install "-j${NPROC}" gd; \
     \
     pecl config-set php_ini "${PHP_INI_DIR}/php.ini"; \
-    \
+    if [[ "${PECL_HTTP_PROXY}" != "" ]]; then \
```

Comment:
the correct way to check for emptiness in bash is using `-n`, see examples in this dockerfile